main.py

```diff
@@ -236,7 +236,9 @@ async def delayed_handle_update(update: Update, timeout: int):
 
     try:
         await dp.feed_update(bot=bot, update=update)
-    except (ConnectionResetError, OSError, ClientOSError, ConnectionError, TelegramServerError, TelegramNetworkError):
+    except (ConnectionResetError, OSError, ClientOSError, ConnectionError, TelegramServerError, TelegramNetworkError) as e:
+        logging.exception(repr(e))
```

Comment:
it's not an exception, use error instead
```suggestion
        logging.error(repr(e))
```